homework-submission/W1-homework/index.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+/**
+ * A function component that inserts a static item in a list
+ * @param {props} param0 The destructured properties description and deadline
+ * @return List item with the values of the properties
+ */
+const StaticTodoItem = ({ description, deadline }) => {
+  return (
+    <li>
+      <h3>{description}</h3>
+      <span>{deadline}</span>
+    </li>
+  );
+};
+
+/**
+ * A function component that inserts a dynamic item in a list
+ * @param {props} param0 The array of todoItems
+ * @return List item with the values of the todoItems of the array if todoItem is not done yet (false)
+ * @return List item with the values of the todoItems of the array if todoItem is done (true)
+ * @return The dynamic list of todoItems
+ */
+const DynamicTodoItem = ({ todoItems }) => {
+  const items = todoItems.map(({ id, description, deadline, done }) => {
+    if (!done) {
+      return (
```

Comment:
Instead of duplicating 4 lines, you could be using `StaticTodoItem` here too, it works!

Reply:
Thanks for this advice ... It has worked also